datarobot_batch_scoring/network/network.py

```diff
@@ -71,8 +75,8 @@ def cacher(*args, **kwargs):
 # There are several reasonable alternatives that are not helpful:
 # - putting a lock around the `session.send` call does not help because
 #   there is only one `.run` call
-# - using multiple `request.Session` objects (one per thread) eliminates
-#   access to thread-pooling
+# - using multiple `request.Session` objects (one per thread)
+#   eliminates access to thread-pooling
 #
 # The use of a cache is useful because it replaces the single lookup (there is
 # only one address we need resolved) with a read operation from then after.
```

Comment:
Hm... Do we still need this monkey patching? If timeout is the root cause, I think in order to simplify things it would be better to remove monkey patching. What do you think?

Reply:
I'll try it.